src/client-app/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Kind of want to rename this to `template.html` to help others understand it's a template used by the `HtmlWebpackPlugin` plugin.